ansible_runner/config/_base.py

```diff
@@ -193,6 +193,17 @@ def _prepare_env(self, runner_mode='pexpect'):
             self.env = {}
             # Special flags to convey info to entrypoint or process in container
             self.env['LAUNCHED_BY_RUNNER'] = '1'
+
+            # A bug in the fuse-overlayfs driver in RHEL < 8.5 causes errors when trying to set extended file attributes.
```

Comment:
I don't want to be too pedantic here, but it's that there is a kernel bug, that results in RHEL8 defaulting to `fuse-overlayfs` and then that `fuse-overlayfs` doesn't support xattrs, which breaks `atomic_move`.

Most details are available at https://www.redhat.com/sysadmin/podman-rootless-overlay

Reply:
No worries. I was trying to explain it succinctly and was not 100% sure I had it right. Thank you for clarifying.